CRM/Contribute/WorkflowMessage/Contribution/BasicContribution.php

```diff
@@ -47,6 +60,37 @@ public function build(array &$example): void {
     $messageTemplate = new $workFlow['class']();
     $this->addExampleData($messageTemplate, $example);
     $example['data'] = $this->toArray($messageTemplate);
+    switch ($example['workflow']) {
+      case 'contribution_online_receipt':
+        $example['asserts'] = [
+          'default' => [
+            ['for' => 'subject', 'regex' => '/Receipt - FIXME Contribution Title - Barbara Johnson/'],
```

Comment:
Is FIXME right?

Reply:
yeah - also copied from Tim's prototype or how it will look when test cover is actually there - aiming to avoid 'FIXME' as a live variable

Reply:
ok